File: server/src/main/java/rs/ac/singidunum/novisad/server/model/PlanZaGodinu.java
package rs.ac.singidunum.novisad.server.model;

import jakarta.persistence.*;

import java.util.Set;

@Entity
public class PlanZaGodinu {

    @Id
    @GeneratedValue
    private Long id;
    private Integer godina;
    @ManyToOne
    private GodinaStudija godinaStudija;
    @OneToMany(mappedBy = "planZaGodinu")
    private Set<PredmetPlanaZaGodinu> predmetiPlanaZaGodinu;

}

File: client/src/app/model/planZaGodinu.ts
import {PredmetPlanaZaGodinu} from "./predmetPlanaZaGodinu";
import {GodinaStudija} from "./godinaStudija";

export interface PlanZaGodinu {
  id: number,
  godina: number,
  brojSemestara: number,
  godinaStudija: GodinaStudija,
  predmetiPlanaZaGodinu: PredmetPlanaZaGodinu[]
}
